3.7/upgrading-starter.md

```diff
@@ -51,21 +51,22 @@ The first step is to install the new ArangoDB package.
 
 **Note:** you do not have to stop the _Starter_ processes before upgrading it.
 
-For example, if you want to upgrade to `3.7.13` on Debian or Ubuntu, either call
+For example, if you want to upgrade to `3.9.2` on Debian or Ubuntu, either call
 
 ```bash
-apt install arangodb=3.7.13
+apt install arangodb=3.9.2
 ```
 
 (`apt-get` on older versions) if you have added the ArangoDB repository. Or
 install a specific package using
 
 ```bash
-dpkg -i arangodb3-3.7.13-1_amd64.deb
+dpkg -i arangodb3-3.9.2-1_amd64.deb
 ```
 
 after you have downloaded the corresponding file from
-[www.arangodb.com/download/](https://www.arangodb.com/download/){:target="_blank"}.
+[www.arangodb.com/download/](https://www.arangodb.com/download/){:target="_blank"}
+or have downloaded the new Enterprise Edition executable.
```

Comment:
The download page leads to both Editions, do we need to mention EE explicitly? Is a .deb executable?